sql/parser/sql.y

```diff
@@ -861,14 +861,14 @@ iso_level:
     // Mapped to the closest supported isolation level.
     $$ = SnapshotIsolation
   }
-| REPEATABLE READ
+| SNAPSHOT
   {
-    // Mapped to the closest supported isolation level.
     $$ = SnapshotIsolation
   }
-| SNAPSHOT
+| REPEATABLE READ
   {
-    $$ = SnapshotIsolation
+    // Mapped to the closest supported isolation level.
```

Comment:
Would be good for any logging or UI display of queries to indicate the precise isolation semantics being used.